src/pages/app/generic/FormView.tsx

```diff
@@ -54,7 +54,7 @@ const FormView: React.FC<Props> = ({
               Create a New {title} by filling out the form below to get started.
               {errors &&
                 JSON.stringify(
-                  ((Object.values(errors)[0] as any) || ({} as any)).message,
```

Comment:
Is change necessary?

Reply:
Oh, not it is not a major change actually, I was testing something there but reverted it back to the original with just spacing changes but I guess it still committed